OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -58,8 +58,7 @@ The Oracle database server container requires custom configuration parameters fo
 
 	`DB_BUNDLE=basic`
 
-	`$ docker run -d --name soadb --network=SOANet -p 1521:1521 -p 5500:5500 -v /u01/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.txt -it --shm-size="8g" container-registry.oracle.com/database/enterprise:12.2.0.1`
-
+	`$ docker run -d --name soadb --network=SOANet -p 1521:1521 -p 5500:5500 --env-file ./db.env.txt -it --shm-size="8g" container-registry.oracle.com/database/enterprise:12.2.0.1`
```

Comment:
You removed the volume from this command which means that if the Database container is removed, the data will disappear. This is probably not what you intended.